sys/suit/transport/worker.c

```diff
@@ -146,6 +146,21 @@ int suit_handle_manifest_buf(const uint8_t *buffer, size_t size)
     return res;
 }
 
+__attribute__((weak))
+void suit_worker_done_cb(int res)
```

Comment:
Maybe provide this as `void suit_worker_done_cb_default(int res)` (not weak) and create a weak alias of that as `suit_worker_done_cb`. That would use the same ROM, but users overwriting the function could also call the default function and do other stuff in addition.

I'm not insisting on this.

Reply:
I mean it's basically just

```C
if (res == 0) {
    pm_reboot();
}
```

I don't think we need a function for that.